src/components/commentary/Commentary.js

```diff
@@ -248,13 +248,31 @@ const Commentary = (props) => {
     str = str.trim();
     return str.startsWith("<br>") ? str.slice(4) : str;
   }, []);
+
+  const getImages = (str, baseUrl) => {
+    const strArr = str.split("<img");
+    const images = [];
+    strArr.forEach((item, i) => {
+      if (i !== 0) {
+        const img = item.split("/>")[0];
+        if (img.indexOf("src") !== 0) {
+          const imgParts = img.split(/\s+/).find((e) => e.includes("src"));
+          const src = imgParts
+            ?.replace("src=", "")
+            ?.replace("base_url", baseUrl);
+          images.push(src);
+        }
+      }
+    });
+    return images;
+  };
+
   const setImages = useCallback(
     (str, imageArr) => {
       const imageObj = { text: str, images: imageArr };
       if (typeof str === "string" && baseUrl !== "") {
         str = str.replaceAll("base_url", baseUrl);
-        const rex = /(?<=src=)"?'?.*?\.(png|jpg)"?'?/g;
-        const images = str.match(rex);
+        const images = getImages(str, baseUrl);
         const newImages =
```

Comment:
We can try to remove the Lookbehind in the regular expressions like this
```suggestion
        const rex = /src\s?=\s?"?'?.*?\.(png|jpg)"?'?/g;
        const images = str
          .match(rex)
          ?.map((a) => a.replaceAll(" ", "").replace("src=", ""));
```